src/Command/MonitorCommand.php

```diff
@@ -123,14 +124,14 @@ private function scheduledTaskFailed(): bool
             'FroshTools.config.monitorTaskGraceTime'
         );
 
-        $date = new \DateTime();
+        $date = new \DateTime('now',new DateTimeZone('UTC'));
```

Comment:
this makes no difference, it's always UTC in shopware

Reply:
@shyim thanks, will change it.